### PR TITLE
Issue_707 NRFU6.11: Updated skip enhancement for nrfu_tests/test_system_hardware_temperature.py 

### PR DESCRIPTION
**Pull request by Dakshatagslab**
# Please include a summary of the changes

**Issue-Id: #707**

**NRFU6.11 Updated skip enhancement for nrfu_tests/test_system_hardware_temperature.py**

# Any specific logic/part of code you need extra attention on

Skip test case with a valid reason and generate both docx and HTML reports correctly.

# Include the Issue number and link

https://github.com/aristanetworks/vane/issues/707

# List/Attach any dependencies/past issues that are required for this change/provide context

# Type of change

- - [ ] Bug fix (non-breaking change which fixes an issue)
- - [ ] New feature (non-breaking change which adds functionality)
- - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- - [ ] This change requires a documentation update
- - [x] Other (please specify)

# Effort required on reviewers end

- - [x] Easy
- - [ ] Medium
- - [ ] Hard 

# How Has This Been Tested?

## Bug fix

    Include before and after snapshots/screenshots/changed logs
    
## New feature

    Ensure current test cases pass and include newer test cases if required
    
# CI pipeline result

- - [ ] Pass
- - [ ] Fail
  
  If it fails, explain the reason and whether or not we should ignore the failure
  
# Verify Documentation Update

    If applicable, ensure documentation has been updated for ReadMe, Getting Started guide, and Style guide
    
# Additional comments

1. Pass report
[pass_report.docx](https://github.com/aristanetworks/vane/files/14692954/pass_report.docx)

2. Skipped reports when the device is vEOS.
[skip_report.docx](https://github.com/aristanetworks/vane/files/14692959/skip_report.docx)

2. Html reports
[html_reports.zip](https://github.com/aristanetworks/vane/files/14692966/html_reports.zip)

### CODE REVIEW

File: nrfu_tests/test_system_hardware_temperature.py [hunk 1]
```diff
@@ -168,8 +168,13 @@ def test_system_temperature_sensors(self, dut, tests_definitions):
             self.output += f"\nOutput of 'show version' command is: \n{self.version_output}"
 
             # Skipping test case if the device is vEOS.
-            if "vEOS" in self.version_output.get("modelName"):
-                pytest.skip(f"{tops.dut_name} is vEOS device, hence test skipped.")
+            model = self.version_output.get("modelName")
+            if "vEOS" in model or "7010" in model:
+                tops.output_msg = f"Skipping test case on {tops.dut_name} as it is {model} device."
```
Comment (dhruvgoel9) [inline nrfu_tests/test_system_hardware_temperature.py:173]:
```suggestion
                tops.output_msg = f"Skipping test case on {tops.dut_name} as it is vEOS device."

```

**Review by dhruvgoel9 — COMMENTED** [review]
Make changes accordingly

**Review by dhruvgoel9 — APPROVED** [review]

File: nrfu_tests/test_system_hardware_temperature.py [hunk 2]
```diff
@@ -168,8 +168,13 @@ def test_system_temperature_sensors(self, dut, tests_definitions):
             self.output += f"\nOutput of 'show version' command is: \n{self.version_output}"
 
             # Skipping test case if the device is vEOS.
-            if "vEOS" in self.version_output.get("modelName"):
-                pytest.skip(f"{tops.dut_name} is vEOS device, hence test skipped.")
+            model = self.version_output.get("modelName")
+            if "vEOS" in model or "7010" in model:
```
Comment (geetanjalimanegslab) [inline nrfu_tests/test_system_hardware_temperature.py:172]: Please attach a skip report where model name is having "7010"

File: nrfu_tests/test_system_hardware_temperature.py [hunk 3]
```diff
@@ -168,8 +168,13 @@ def test_system_temperature_sensors(self, dut, tests_definitions):
             self.output += f"\nOutput of 'show version' command is: \n{self.version_output}"
 
             # Skipping test case if the device is vEOS.
-            if "vEOS" in self.version_output.get("modelName"):
-                pytest.skip(f"{tops.dut_name} is vEOS device, hence test skipped.")
+            model = self.version_output.get("modelName")
+            if "vEOS" in model or "7010" in model:
+                tops.output_msg = f"Skipping test case on {tops.dut_name} as it is vEOS device."
```
Comment (geetanjalimanegslab) [inline nrfu_tests/test_system_hardware_temperature.py:173]:
```suggestion
                tops.output_msg = f"{tops.dut_name} is {model} device, hence test skipped."

```

**Review by geetanjalimanegslab — APPROVED** [review]

**Review by rewati-arista — APPROVED** [review]

**Review by shachiagarwal — APPROVED** [review]

**Pull request merged**